### PR TITLE
Elaborated Context and Container Diagrams

### PR DESCRIPTION
**Pull request by neilkidd**
This PR adds further information to the Context and container diagrams following meetings with:

- Luca Lischetti
- Tuomo Karki
- Daro Clark

### CODE REVIEW

File: images/system-container.iuml [hunk 1]
```diff
@@ -11,25 +11,44 @@ LAYOUT_AS_SKETCH()
...
+  System_Ext(google_sheets, "Google Sheets", "Form submission data")
+  System_Ext(google_docs, "Google Docs", "Generated for viewing (AppScript)")
+}
 
 System_Boundary(system, "Social Care") {
 
-    Container(social_care_front_end, "Social Care Front End", "Lambda, React - TypeScript", "Allows practitioners to edit case information.")
-    Container(social_care_service_api, "Social Care Case Viewer Service API", "Lambda, .Net - C#", "Provides some abstraction")
-    Container(mosaic_historic_data_api, "Residents Social Care Platform API",  "Lambda, .Net - C#", "Serves historic read-only case and visit data")
+  Container(social_care_front_end, "Social Care Front End", "Lambda, React - TypeScript", "Allows practitioners to edit case information.")
```
Comment (sirLisko) [inline images/system-container.iuml:27]:
```suggestion
  Container(social_care_front_end, "Social Care Front End", "Lambda, Next.js (React) - TypeScript", "Allows practitioners to edit case information.")
```
I would add `Next.js` that is the framework that we are using, as it's with next that we proxy the APIs.

Also, I don't know if it's worth adding it, but we are already giving the practitioners the ability to add people to the system and in this sprint, we should (hopefully) be able to edit them.

Comment (neilkidd) [reply images/system-container.iuml]: I've added `Next.js` as per you suggestion - thanks :)

I'm not convinced the specific functionalities are useful at this level - so won't add it for now.

**Review by sirLisko — APPROVED** [review]
Looks good to me :)

**Review by chubberlisk — APPROVED** [review]
LGTM - a number of things I didn't know / slipped out of my mind!

**Pull request merged**